src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> age >= 30 && age <= 45)
```

Comment:
필터 내 조건이 복잡하다면
추가 체이닝하는 것도 고려해 보세요!
```suggestion
                .filter(age -> age >= 30)
                .filter(age -> age <= 45)
```